zipline/finance/trading.py

```diff
@@ -204,17 +204,18 @@ def normalize_date(self, test_date):
         return pd.tseries.tools.normalize_date(test_date)
 
     def utc_dt_in_exchange(self, dt):
-        return pd.Timestamp(dt).tz_convert(self.exchange_tz)
+        return pd.Timestamp(dt, tz='UTC').tz_convert(self.exchange_tz)
```

Comment:
@warren-oneill from reading the name, I think the contract of this method is that it expects to be passed a UTC dt.  Is there an existing location where we're passing a possibly not UTC dt here?